src/components/common/table/TableDensitySettingsButton.tsx

```diff
@@ -62,14 +73,12 @@ const TableDensitySettingsButton = () => {
         />
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 66]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Could you update the tooltip to just "Settings"?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2771?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->